README.md

```diff
@@ -16,29 +16,42 @@ To install `outpack.orderly`:
 remotes::install_github("mrc-ide/outpack.orderly", upgrade = FALSE)
 ```
 
-## Usage
+Recommended usage is from docker.
+
+## Usage from docker
 
 ```
 
 Usage:
-  orderly2outpack.R <src> <dest> [--once]
+  mrcide/outpack.orderly <src> <dest> [--once] [--custom=<cron-schedule> ]
 
 Options:
-  --once      Perform migration once.
+  --once                      Perform migration once.
+  --custom=<cron-schedule>    Schedule migration using cron.
 
 ```
 
-## Usage from docker
-
 You will have to first mount the `orderly` and `outpack` directories as volumes.
-`orderly` can be readonly. 
+`orderly` can be readonly. To have the job run as a non-root user (to avoid created files being owned by root),
+pass the desired uid as an env var, as shown.
 
 ```
 docker run -v /orderly/path:/orderly:ro \
            -v /outpack/path:/outpack \
+           --env USER_ID=$UID \
            mrcide/outpack.orderly /orderly /outpack --once
 ```
 
+If running on a schedule, you most likely want to run in detached mode:
+
+```
+docker run -v /orderly/path:/orderly:ro \
+           -v /outpack/path:/outpack \
+           --env USER_ID=$UID \
+           -d \
+           mrcide/outpack.orderly /orderly /outpack --custom="* * * * *"
```

Comment:
does this command work in practice? There are one or two places there in which that quoting could be stripped and something will interpret `*` as a shell glob rather than a literal star

Reply:
It does work, as per the test